sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentServiceRequest.java

```diff
@@ -713,40 +714,57 @@ public static RxDocumentServiceRequest createFromName(
         );
     }
 
+    // The path will be in the form of
+    // [/]{resourceType/resourceId}+[/] or
+    // [/]{resourceTyp]/resourceI]}+/resourceType[/]
+    //
+    // Expected output in both cases are [resourceId]
     private static String extractIdFromUri(String path) {
```

Comment:
Please consider adding unit tests for `RxDocumentServiceRequest#extractIdFromUri()`
or unit tests for `RxDocumentServiceRequest` which captures the relevant code change.

Reply:
Its actually dead code